models/cow_protocol/ethereum/cow_protocol_ethereum_schema.yml

```diff
@@ -293,12 +293,14 @@ models:
         name: buy_quote
       - &sell_quote
         name: sell_quote
-      - *slippage_bips
-      - *usd_value
-      - &surplus_atoms
-        name: surplus_atoms
-      - &surplus_percentage
-        name: surplus_percentage
-      - &surplus_usd
-        name: surplus_usd
+      - &slippage_tolerance_bips
+        name: slippage_tolerance_bips
+      - &trade_usd_value
+        name: trade_usd_value
+      - &slippage_atoms
+        name: slippage_atoms
+      - &slippage_percentage
+        name: slippage_percentage
+      - &slippage_usd
+        name: slippage_usd
```

Comment:
nit: since the table name contains the term slippage, its pretty well known that all the values here are referring to slippage. Just I would think these terms could be `tolerance_bips`, `amount_atoms`, `amount_usd` etc...